plugins/pinecone.yaml

```diff
@@ -0,0 +1,103 @@
+version: 0.0.1
+title: Pinecone Metrics
+description: Metrics receiver for Pinecone
+parameters:
+  - name: environment
+    description: Environment of your project  (example us-east4-gcp)
+    type: string
+    required: true
+  - name: api_key
+    description: The api key of your pinecone environment
+    type: string
+    required: true
+  - name: scrape_interval
+    description: Time in between every scrape request
+    type: string
+    default: 60s
+template: |
+  receivers:
+    prometheus/pinecone:
+      config:
+        scrape_configs:
+          - job_name: 'pinecone'
+            scrape_interval: '{{ .scrape_interval }}'
+            authorization:
+              credentials: '{{ .api_key }}'
+            scheme: https
```

Comment:
we're forcing TLS here will it always work or is there a locally hosted version of pinecone where they don't require TLS?

Reply:
I could be wrong, but I believe Pinecone is a saas, which would always have TLS.